src/Init/Data/BitVec/Lemmas.lean

```diff
@@ -2837,13 +2826,21 @@ theorem toInt_intMin {w : Nat} :
     rw [Nat.mul_comm]
     simp [w_pos]
 
+@[simp]
+theorem not_intMin {w : Nat} :
+    ¬ intMin w = 0#w := by sorry
```

Comment:
Is this true for `w = 0`.

Reply:
I'm still a bit confused about what happens if `w=0`. By the def `intMin w = twoPow w (w - 1) = BitVec w := 1#w <<< (w - 1)`, so I would expect this to hold. But I am not sure yet.

Reply:
I guess I can check with `#eval` as well

Reply:
actually this theorem makes no sense at all

Reply:
Why? It could could say that intMin is not 0#w for any w > 0, no? Not sure if this is useful, though.

Reply:
It might be useful to keep in mind that `BitVec 0` has exactly *one* inhabitant. Any method of constructing a `BitVec 0` will thus give a result equal to `0#0`

Reply:
I ended up not using it at all, happy to add it if it can be useful though. 

Also, @alexkeizer I just spent some hours trying to understand this thing that `1#0` is equal to `0#0`. The concept is very clear and sensible, however I am struggling to deal with these objects in the proofs. Is there a general desirable approach to handle these cases? While trying to prove `msb_neg` I ended up having in/equalities dealing with them quite a few times and I struggled to get something reasonable out of it.